src/test/java/com/example/gradecalculator/entities/SubjectTest.java

```diff
@@ -10,16 +9,5 @@ public void testGettersAndSetters() {
 
         Subject subject = new Subject();
 
-
-        subject.setId(1L);
-        subject.setName("Subject 1");
-        subject.setDescription("Description 1");
-        subject.setCreditValue(3);
-
-
-        Assertions.assertEquals(1L, subject.getId());
-        Assertions.assertEquals("Subject 1", subject.getName());
-        Assertions.assertEquals("Description 1", subject.getDescription());
-        Assertions.assertEquals(3, subject.getCreditValue());
```

Comment:
now its an empty test - better to remove it entirely. But why was the test removed?

Reply:
I have now written the "SubjectTest" new, I also have spoken to Dianna and she didn't knew why she has deleted thees parts of the test

Reply:
Well there is a catch ;)
Subject could just use Lombok getters and setters and then there would be no need to test those